src/components/dev/LocalizationReport.js

```diff
@@ -96,17 +105,22 @@ class LocalizationReport extends Component {
   
   setFilter = (e, { value }) => { this.setState({ filter: value }); };
  
-  filterResults = (results, filter) => {
-    if (filter === 'all') { 
+  filterResults (results, filter) {
+    if (filter === `all`) { 
       return results;
     }
-    let pass = filter === 'true' ? true : false;
+
+    let pass = true;
```

Comment:
In this specific case, this could instead be replaced with `const [or let] pass = filter === 'true';`. Is this intentionally avoiding that construct for clarity?

Reply:
No, it's not, lol. Just blanked, thanks!